src/details/ArborX_PairValueIndex.hpp

```diff
@@ -56,7 +56,9 @@ class AttachIndices
   auto size() const { return _data.size(); }
 };
 
-template <typename Values, typename Index = unsigned>
+// Make sure the default Index matches the default in PairValueIndex
```

Comment:
I don't understand what you mean by make sure?
Should that be a static assertion?

Reply:
They must actually match when constructing hierarchy. However, we cannot really enforce it, imho, as user is responsible for specifying both. The next best thing is to at least make sure that the default values for both match.

Reply:
Is this about making sure that the specified default (`unsigned`) matches the default from `PairValueIndex`?

Reply:
Yes.